src/InnerImageZoom/InnerImageZoom.js

```diff
@@ -119,8 +117,11 @@ const InnerImageZoom = ({
 
   const handleDragMove = useCallback((e) => {
     e.stopPropagation();
-    let left = (e.pageX || e.changedTouches[0].pageX) - imgProps.current.offsets.x;
-    let top = (e.pageY || e.changedTouches[0].pageY) - imgProps.current.offsets.y;
+    const pageX = typeof e.pageX === 'number' ? e.pageX : e.changedTouches[0].pageX;
+    const pageY = typeof e.pageY === 'number' ? e.pageY : e.changedTouches[0].pageY;
+    //console.log(`Move pageX: ${pageX} pageY: ${pageY}`);
```

Comment:
This line could also probably be deleted.

Reply:
Yes, not needed.